frappe/website/doctype/blog_post/blog_post.py

```diff
@@ -54,18 +56,29 @@ def get_context(self, context):
 		context.updated = global_date_format(self.published_on)
 
 		if self.blogger:
-			context.blogger_info = frappe.get_doc("Blogger", self.blogger).as_dict()
+			context.blogger_info = _(frappe.get_doc("Blogger", self.blogger).as_dict())
 
-		context.description = self.blog_intro or self.content[:140]
+		context.description = _(self.blog_intro) or _(self.content[:140])
 
 		context.metatags = {
-			"name": self.title,
-			"description": context.description,
+			"name": _(self.title),
+			"description": _(context.description),
 		}
 
 		if "<!-- markdown -->" in context.content:
 			context.content = markdown(context.content)
 
+		is_jinja = "<!-- jinja -->" in context.content
```

Comment:
`is_jinja = "<!-- jinja -->" in context.content or ("{{" in context.content)`

make it `frappe.utils.is_jinja`